cmd/nerdctl/run_network_linux_test.go

```diff
@@ -396,6 +397,15 @@ func TestRunPort(t *testing.T) {
 
 }
 
+func TestRunWithInvalidPortThenCleanUp(t *testing.T) {
+	// docker does not set label restriction to 4096 bytes
+	testutil.DockerIncompatible(t)
+	t.Parallel()
+	base := testutil.NewBase(t)
+	containerName := testutil.Identifier(t)
+	base.Cmd("run", "--rm", "--name", containerName, "-p", "22200-22299:22200-22299", testutil.CommonImage).AssertCombinedOutContains(errdefs.ErrInvalidArgument.Error())
```

Comment:
We should test that we can reuse `containerName` for another container

Reply:
fixed

Reply:
Doesn't seem fixed